src/applications/gi/utils/helpers.js

```diff
@@ -46,13 +46,15 @@ export const phoneInfo = (areaCode, phoneNumber) => {
   return providerPhone;
 };
 
-export const snakeCaseKeys = query =>
-  Object.keys(query).reduce(
-    (queryParams, key) => ({
+export const rubyifyKeys = query =>
```

Comment:
Could you put a function comment for this describing how it works? When I look at the tests, it seems pretty apparent to me, but it'd be good to have a little description right here too. Bonus points for adding in _why_ it does it this way. :smile: